hivemind/dht/routing.py

```diff
@@ -10,7 +10,7 @@
 from typing import Any, Dict, List, Optional, Sequence, Set, Tuple, Union
 
 from hivemind.p2p import PeerID
-from hivemind.utils import MSGPackSerializer, get_dht_time
+from hivemind.utils import DHTExpiration, MSGPackSerializer, get_dht_time
```

Comment:
nit: DHTExpiration is not needed in `routing.py`. It would be better to remove this change and import `DHTExpiration` directly from `hivemind.utils` in `dht/__init__.py`.